src/components/Header/Header.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react"
+import { useSelector } from "react-redux"
+import HeaderAuth from "./HeaderAuth"
+import HeaderUser from './HeaderUser'
+
+const Header = () => {
+
+    const stateUsInfouserData = useSelector(state => state.userInfo.userData)
```

Comment:
IMHO this var isnt readable at all - `stateUsInfouserData`. I would name it userData

Reply:
Of course - I will replace it with userData) Thanks for the advice)